CMakeLists.txt

```diff
@@ -876,11 +880,18 @@ function(build_loadable_extension_directory NAME OUTPUT_DIRECTORY EXTENSION_VERS
       COMMAND emcc $<TARGET_FILE:${TARGET_NAME}> -o $<TARGET_FILE:${TARGET_NAME}>.wasm -O3 -sSIDE_MODULE=2 -sEXPORTED_FUNCTIONS="_${NAME}_init,_${NAME}_version" ${WASM_THREAD_FLAGS} ${TO_BE_LINKED}
       )
   endif()
+
+  if (${ABI_TYPE} STREQUAL "CPP")
```

Comment:
In a follow-up PR can we rename this to `CPP_DEPRECATED` to indicate we plan to unify this under `C_STRUCT` in the future?